sdk/appconfiguration/app-configuration/src/appConfigurationClient.ts

```diff
@@ -486,3 +494,30 @@ export class AppConfigurationClient {
     });
   }
 }
+
+function getAppConfigurationOptions(
+  baseUri: string,
+  syncTokens: SyncTokens,
+  deserializationContentTypes: DeserializationContentTypes
```

Comment:
If you decide not to update core-http to get this interface, you could get the type doing something like:
```
contentTypes: typeof deserializationContentTypes 
```

Reply:
"Filed away for future reference" ;)